includes/admin/views/tmpl-order-item.php

```diff
@@ -20,7 +20,7 @@
 <td class="has-row-actions column-primary">
 	<div class="removable">
 		<# if ( true === data.state.isAdding ) { #>
-		<button class="button-link delete">
+		<button class="button-link delete" aria-label="removeItem">
```

Comment:
I'm curious about why you chose to use an `aria-label` instead of adding text to the button and hiding it with `screen-reader-text`? I believe your approach is valid, so I don't think it needs to change, but is a string which should be translatable and which may be read by screen reader users (as an example), so it should be internationalized and a phrase, please.

Reply:
I did a bunch of Googling and never came across anyone suggesting using a label, though it seems obvious now.  As that is what we use for other hidden labels, I'm going to change it!

Reply:
It didn't even dawn on me to use a label.  I Googled a bunch and `aria-label` seemed to be the most standard and [this](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html) really decided for me.  I can and will, however, change it to a string that can be translated.